manifests/init.pp

```diff
@@ -49,6 +49,7 @@
   $service_name       = 'mcollective',
   $server_package     = 'mcollective',
   $ruby_stomp_package = 'ruby-stomp',
+  $ruby_interpreter   = '/usr/bin/env ruby',
```

Comment:
in manifests/server/config/factsource/yaml.pp there is the $ruby_shebang_path set based on $::is_pe, so setting it to something for pe, hardcoded, or now after the patch, to $ruby_interpreter.
That should be pe compatible. The spec tests, I added, also test for the is_pe case.